### PR TITLE
Added properties Window.width and Window.height

### PR DESCRIPTION
**Pull request by clsource**
For completeness.

In dome.c

```c
internal void
WINDOW_getWidth(WrenVM* vm) {
  ENGINE* engine = (ENGINE*)wrenGetUserData(vm);
  int width = 0;
  SDL_GetWindowSize(engine->window, &width, NULL);
  wrenSetSlotDouble(vm, 0, width);
}

internal void
WINDOW_getHeight(WrenVM* vm) {
  ENGINE* engine = (ENGINE*)wrenGetUserData(vm);
  int height = 0;
  SDL_GetWindowSize(engine->window, NULL, &height);
  wrenSetSlotDouble(vm, 0, height);
}
```

Example code
```js
System.print("%(Window.width) %(Window.height)")
```

### CODE REVIEW

**Comment by avivbeeri** [comment]
This looks fine. We am wondering if we should implement distinct setters for these fields also. At the moment, `Window.resize(w, h)` is used for that, but people might expect it to be possible to set the window size using individual dimensions.

The `resize` API is like that because it mirrors the `Canvas.resize` operation (which is intentionally awkward, because resizing the Canvas is not a cheap operation at all).

Secondly, these fields will need documenting in the `docs/modules/dome.md` file.

**Comment by avivbeeri** [comment]
Please could you grant me permission to commit to your this branch in your fork of DOME, so that I can push a change to the documentation which describes the `width` and `height` properties?

See here: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork

**Comment by clsource** [comment]
No worries I will make them :)

File: docs/modules/dome.md [hunk 1]
```diff
@@ -1,41 +1,57 @@
...
 #### `static title: String`
+
 This allows you to set and get the title of the DOME window.
+
 #### `static vsync: String`
+
 Setting this to true will make the renderer wait for VSync before presenting to the display. Changing this value is an expensive operation and so shouldn't be done frequently.
 
-### Static Methods
+#### `static height: Number`
```
Comment (avivbeeri) [inline docs/modules/dome.md:45]: Please move the `height` entry up to after `fullscreen`, I try to keep these in alphabetical order. :)

Comment (clsource) [reply docs/modules/dome.md:45]: ok

**Comment by avivbeeri** [comment]
This looks good! Time for merging :)

**Pull request merged**